pkg/fs/fs_windows.go

```diff
@@ -0,0 +1,52 @@
+package fs
+
+import "os"
+
+func SyncDir(dirName string) error {
+	return nil
+}
+
+// RenameFileWithReplacement will replace any existing file at newpath with the contents
+// of oldpath.
+//
+// If no file already exists at newpath, newpath will be created using the contents
+// of oldpath. If this function returns successfully, the contents of newpath will
+// be identical to oldpath, and oldpath will be removed.
+func RenameFileWithReplacement(oldpath, newpath string) error {
+	if _, err := os.Stat(newpath); err == nil {
```

Comment:
Would it be safer to rename the existing `newpath` to a temporary file before renaming oldpath, and then doing the removal last?

Reply:
I'm not exactly sure what you are asking, but using `RenameFileWithReplacement` is safe if you expect and are planning for any old contents of `newpath` to be truncated and lost. The `RenameFile` func exists as a tool to ensure you do not lose any data accidentally.